tests/integration/test_users.py

```diff
@@ -140,6 +140,17 @@ def test_user_roles(client, auth_headers):
     assert response.status_code == 200
 
 
+def test_user_no_roles(client, auth_headers, test_database):
```

Comment:
We followed the way another integration test was written and had the assertion check not only users seeded in the database but also a user added via the test fixtures. This felt strange, as it seemed like the test should have only checked the seeded database OR the test database, not both. If there's not a particular reason to check both I'd like to just check the ones in the seeded database.